packages/calypso-build/webpack.config.js

```diff
@@ -99,8 +100,8 @@ function getWebpackConfig(
 				extractComments: false,
 				terserOptions: {
 					ecma: 5,
-					safari10: true,
-					mangle: true,
+					safari10: ! isDesktop,
```

Comment:
I would pass in the options from `client/webpack.config.js` instead. Note that it's already doing that for `mangle`, and you'd only have to do the same for `safari10`.

Reply:
Actually, on second thought, I would drop this altogether. Changing this file shouldn't affect Calypso builds at all, since Calypso has its own webpack config. And changing this in `client/webpack.config.js` will affect how the value gets calculated.